news/_posts/public-oss-demo/2021-04-08-public-oss-demo-3.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title:  "PrestaShop Public OSS Demo - March 2021 session"
+subtitle: "We held the third session of the open source project's public demo"
+date:   2021-04-09 12:00:00
+authors: [ MathieuFerment ]
+icon: icon-video-camera
+tags: [contribute, event, community]
+---
+
+On the 31st of March, the maintainers team held again a video conferencing for the project public demonstrations session, streamed live on [YouTube](https://youtu.be/UJ_XQs8trs4).
+
+## The third Public open source demonstrations agenda
+
```

Comment:
```suggestion
```

Reply:
Do we need this title? I find it kind of repetitive

Reply:
I added it on purpose to make it repetitive and keep a common format with all "blog posts about a demo that happened".

We can remove it to lighten it :)

Reply:
Removed https://github.com/PrestaShop/prestashop.github.io/pull/878/commits/4938ac067e20450fd468403d238490b93be62a53